src/encoded/static/components/filegallery.js

```diff
@@ -2500,40 +2576,20 @@ const AnalysesSelector = ({ analyses, selectedAnalysesIndex, handleAnalysesSelec
     });
 
     /**
-     * Selects certain awards as the default if listed.
+     * Select default for "Choose Analysis".
     */
     useMount(() => {
-        const awardSuffix = ['ENCODE4', 'ENCODE3', 'ENCODE2', 'Mixed', 'Lab custom']; // ordered in decreasing precedence
-        const analysesAwarded = analyses.filter((analysis) => awardSuffix.some((award) => (analysis.title || '').includes(award)));
-
-        // a bit hacky, removes text and use number to determine sorting
-        const sortedAnalysesAwarded = analysesAwarded.sort((a, b) => {
-            // remove text
-            const numbersInTitle1 = a.title.replace(/\D/g, '');
-            const numbersInTitle2 = b.title.replace(/\D/g, '');
-
-            // add numbers so ones with version have the same number of digits as non-versioned ones
-            const number1 = numbersInTitle1.padEnd ? numbersInTitle1.padEnd(8, 0) : numbersInTitle1;
-            const number2 = numbersInTitle2.padEnd ? numbersInTitle2.padEnd(8, 0) : numbersInTitle2;
+        const awardSuffixes = ['ENCODE4', 'ENCODE3', 'ENCODE2', 'Mixed', 'Lab custom']; // ordered in decreasing precedence
+        const statuses = ['released', 'in progress', 'archived', 'deleted', 'revoked']; // ordered in decreasing precedence
```

Comment:
@forresttanaka  I tried doing

```
     const accessLevel = sessionToAccessLevel(context.session, context.session_properties);
        const statuses = getObjectStatuses('Analysis', accessLevel); // ordered in decreasing precedence
```

The issue is that I did not get all needed. Like for [http://encd-5820-5283310b1-phil.demo.encodedcc.org/experiments/ENCSR000CNC/](http://encd-5820-5283310b1-phil.demo.encodedcc.org/experiments/ENCSR000CNC/), I, did not get back archived even though an analyses record has status of archived.

Reply:
You’re using `standardStatuses` which doesn’t include archived. Analysis statuses don’t match any existing status scheme, so you need to make a custom one, though in my example I don’t know exactly what access levels each one has in Analysis objects, nor do I know what order they should be, but this should come close to correct:

```
    AccessKey: {
        external: [
            'current',
        ],
        administrator: [
            'deleted',
        ],
    },
    Analysis: {
        external: [
            'released',
            'archived',
            'revoked',
        ],
        administrator: [
            'in progress',
            'deleted',
        ].
    },
    AnalysisStepRun: standardStatuses,
    AnalysisStepVersion: standardStatuses,